### ISSUE TITLE
add support for SVG tags 

### ISSUE THREAD

**Issue by hosamkora** [daohoangson/flutter_widget_from_html#132]
adding support for **SVG** tag using 
_https://pub.dev/packages/flutter_svg_  
or any implementation suits your needs

**Comment by daohoangson** [comment]
I think this may work.

**Comment by Nijinsha** [comment]
any updates on this?

**Comment by daohoangson** [comment]
I have a dev version up which includes experimental support for SVG. You can try it by updating your pubspec to use this version:

```yaml
dependencies:
  flutter_widget_from_html: ^0.4.0-rc.2020021401
```

Example code

```dart
import 'package:flutter/material.dart';
import 'package:flutter_widget_from_html/flutter_widget_from_html.dart';

const kHtml = """
<p>Hello.</p>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 166 202">
    <defs>
        <linearGradient id="triangleGradient">
            <stop offset="20%" stop-color="#000000" stop-opacity=".55" />
            <stop offset="85%" stop-color="#616161" stop-opacity=".01" />
        </linearGradient>
        <linearGradient id="rectangleGradient" x1="0%" x2="0%" y1="0%" y2="100%">
            <stop offset="20%" stop-color="#000000" stop-opacity=".15" />
            <stop offset="85%" stop-color="#616161" stop-opacity=".01" />
        </linearGradient>
    </defs>
    <path fill="#42A5F5" fill-opacity=".8" d="M37.7 128.9 9.8 101 100.4 10.4 156.2 10.4"/>
    <path fill="#42A5F5" fill-opacity=".8" d="M156.2 94 100.4 94 79.5 114.9 107.4 142.8"/>
    <path fill="#0D47A1" d="M79.5 170.7 100.4 191.6 156.2 191.6 156.2 191.6 107.4 142.8"/>
    <g transform="matrix(0.7071, -0.7071, 0.7071, 0.7071, -77.667, 98.057)">
        <rect width="39.4" height="39.4" x="59.8" y="123.1" fill="#42A5F5" />
        <rect width="39.4" height="5.5" x="59.8" y="162.5" fill="url(#rectangleGradient)" />
    </g>
    <path d="M79.5 170.7 120.9 156.4 107.4 142.8" fill="url(#triangleGradient)" />
</svg>

<p>How are you?</p>
""";

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) => MaterialApp(
        home: Scaffold(
          appBar: AppBar(title: Text('issue 132')),
          body: HtmlWidget(kHtml),
        ),
      );
}
```

<img width="508" alt="Screen Shot 2020-02-14 at 9 25 43 AM" src="https://user-images.githubusercontent.com/239336/74496018-0298ed80-4f0c-11ea-9d06-451498159b8b.png">

**Comment by daohoangson** [comment]
Does the RC version work for you? @hosamkora @Nijinsha

**Comment by daohoangson** [comment]
This has been merged into master (see #133). It will be available in the next release.
